app/views/alchemy/_preview_mode_code.html.erb

```diff
@@ -0,0 +1,4 @@
+<% if @preview_mode %>
+  <%= javascript_tag("Alchemy = { locale: '#{session[:alchemy_locale]}' };") +
+    javascript_include_tag("alchemy/preview") %>
```

Comment:
Good catch. I just copied the content over to the partial without really looking at it.
Done and pushed.